examples/tutorial/tutorial_notebooks/introduction/exercise_1.ipynb

```diff
@@ -204,13 +206,129 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "**Please move on to Exercise 2 when you are ready**"
+    "### Modin on OmniSci\n",
+    "\n",
+    "Let's run examples to run using on `OmniSci`. For this need activate ``Omnisci`` in `StorageFormat` from `modin.config`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Note: Do not change this code!\n",
+    "import numpy as np\n",
+    "import pandas\n",
+    "import subprocess\n",
+    "import sys\n",
+    "import modin\n",
+    "import modin.config as cfg\n",
+    "cfg.StorageFormat.put('omnisci')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Implement your answer here. You are also free to play with the size\n",
+    "# and shape of the DataFrame, but beware of exceeding your memory!\n",
+    "\n",
+    "import pandas as pd\n",
+    "\n",
+    "frame_data = np.random.randint(0, 100, size=(2**10, 2**5))\n",
+    "df = pd.DataFrame(frame_data)\n",
+    "\n",
+    "# ***** Do not change the code below! It verifies that \n",
+    "# ***** the exercise has been done correctly. *****\n",
+    "\n",
+    "try:\n",
+    "    assert df is not None\n",
+    "    assert frame_data is not None\n",
+    "    assert isinstance(frame_data, np.ndarray)\n",
+    "except:\n",
+    "    raise AssertionError(\"Don't change too much of the original code!\")\n",
+    "assert \"modin.pandas\" in sys.modules, \"Not quite correct. Remember the single line of code change (See above)\"\n",
+    "\n",
+    "import modin.pandas\n",
+    "assert pd == modin.pandas, \"Remember the single line of code change (See above)\"\n",
+    "assert hasattr(df, \"_query_compiler\"), \"Make sure that `df` is a modin.pandas DataFrame.\"\n",
+    "\n",
+    "print(\"Success! You only need to change one line of code!\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the first 10 lines.\n",
+    "df.head(10)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Print the DataFrame.\n",
+    "df"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Free cell for custom interaction (Play around here!)\n",
+    "df.add_prefix(\"col\")"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.count()"
```

Comment:
Looks like `count` is broken now in OmniSci:
```
import numpy as np
import modin.pandas as pd
import modin.config as cfg

cfg.StorageFormat.put("omnisci")

frame_data = np.random.randint(0, 100, size=(2**10, 2**5))
df = pd.DataFrame(frame_data)

print(df.count())
```
Output:
```
  File "pyarrow/table.pxi", line 2153, in pyarrow.lib.Table.rename_columns
  File "stringsource", line 15, in string.from_py.__pyx_convert_string_from_py_std__in_string
TypeError: expected bytes, int found
```

Reply:
Interesting... I believe we have tests for count with Omnisci engine...

Reply:
It's a known problem, it's caused by non-string column labels, I haven't found an issue for this though...

Reply:
Although it's not stated anywhere, but the OmniSci backend doesn't support non-string column labels. The problem should go away once you specify valid labels for the frame.

Reply:
Ok, i see. Looks like issue here can be eliminated by replacement `df.add_prefix("col1")` -> `df = df.add_prefix("col")`.

Reply:
Tests with ``count`` was deleted

Reply:
@dchigarev, why doesn't `df.add_prefix("col1")` work in-place for OmniSci?

Reply:
IDK, it works on my machine (c)
```python
>>> df = pd.DataFrame([[1, 2], [3, 4]])
>>> df
   0  1
0  1  2
1  3  4
>>> df.columns
RangeIndex(start=0, stop=2, step=1)
>>> df.count()
TypeError: expected bytes, int found
>>> df.add_prefix("col1").count()
col10    2
col11    2
dtype: int32
>>> df.add_prefix("col").count()
col0    2
col1    2
dtype: int32
```
Both of the `add_prefix` should work...

Reply:
Ok, thanks!